keith-vscode/src/table/table-webview.ts

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-promise-executor-return */
+/* eslint-disable no-console */
+/* eslint-disable no-return-assign */
+/* eslint-disable prefer-template */
+/* eslint-disable prettier/prettier */
```

Comment:
We would like to use the configured eslint/prettier configuration, disabling should only be done for good (documented) reasons or the configuration should be changed if we really do not want to enforce some of the configured behavior.
Do you have the prettier extension installed in your VS Code? That allows to show the errors while writing the code, not only when trying to build it.